cli/tests/pcluster/schemas/test_schema_validators.py

```diff
@@ -645,6 +645,16 @@ def test_budget_notification_schema_validators(
         (None, "alias@amazon.com", None),
         ("EMAIL", "alias@amazon.com", None),
         ("SNS", "arn:aws:sns:us-east-1:444455556666:MyTopic", None),
+        ("SNS", "arn:sns:us-east-1:56986798:topic", "The specified valued for Address is not a valid SNS topic arn."),
+        (
+            "SNS",
+            " arn:aws:sns:us-east-1:444455556666:MyTopic",
+            "The specified valued for Address is not a valid SNS topic arn.",
+        ),
+        ("SNS", "12321412:213432", "The specified valued for Address is not a valid SNS topic arn."),
+        ("EMAIL", "myemail.com", "The specified value for Address is not a valid Email address."),
+        ("EMAIL", "amazonemail@amazon.comcom", "The specified value for Address is not a valid Email address."),
+        ("EMAIL", "email@gmail.amazom.com", "The specified value for Address is not a valid Email address."),
```

Comment:
Should it be a valid email address? Emails address can have sub subdomain. Let's double check the regex is correct

Reply:
Fixed!